### ISSUE TITLE
Modify process for loading `centreline_volumes` to only load "clean" data

### ISSUE THREAD

**Issue by jwrcoleman** [CityofToronto/bdit_volumes#26]
Edit process to first clean traffic data stored in `det` and `cnt_det`  (using temporary tables) prior to populating `centreline_volumes`.

**Comment by aharpalaniTO** [comment]
Related to, and being developed in parallel to, #23. Overall these two issues consist of two steps:

1. Report identifying bad count data to set to Trevor (for further investigation / clean-up of FLOW.
2. Cleaning our own database to feed clean dataset into model prior to FLOW update.

**Comment by aharpalaniTO** [comment]
Closing as near duplicate of issues #23 and #27
